repak/src/pak.rs

```diff
@@ -83,28 +83,58 @@ fn decrypt(key: Option<&aes::Aes256>, bytes: &mut [u8]) -> Result<(), super::Err
 }
 
 impl PakReader {
-    pub fn new_any<R: Read + Seek>(
+    #[cfg(not(feature = "encryption"))]
```

Comment:
that's what I was addressing with this since calling new_any_with_key with none is the same as new_any
>  also had the idea of making new_any_with_key accept an aes key and not excluding new_any when encryption is enabled but a lot of the time the user will probably be using options so not 100% sure on that

Reply:
there can be a convenience function with options too :)